python/mleap/sklearn/feature_extraction/text.py

```diff
@@ -15,33 +15,64 @@
 # limitations under the License.
 #
 
-from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import CountVectorizer, TfidfVectorizer
+from mleap.sklearn.pipeline import SimpleSerializer as PipelineSerializer
 from mleap.bundle.serialize import MLeapSerializer
 import uuid
 
 
+class ops(object):
+    def __init__(self):
+        self.COUNT_VECTORIZER = 'tokenizer'
+        self.TF_IDF_VECTORIZER = 'pipeline'
+
+
+ops = ops()
+
+
 def serialize_to_bundle(self, path, model_name):
-    serializer = SimpleSerializer()
```

Comment:
I believe this should be SimpleSerializer so that CountVectorizer would work on its own and not just part of the tf-idf.

Reply:
Nice catch. Will fix, and write a test for CountVectorizer.